code/modules/assembly/signaler.dm

```diff
@@ -29,6 +29,18 @@ GLOBAL_LIST_EMPTY(remote_signalers)
 	. = ..()
 	. += "The power light is [receiving ? "on" : "off"]"
 
+/obj/item/assembly/signaler/AltClick(mob/user)
+	activate()
+
+/obj/item/assembly/signaler/attackby(obj/item/W, mob/user, params)
+	if(issignaler(W))
+		var/obj/item/assembly/signaler/signaler2 = W
+		if(secured && signaler2.secured)
+			code = signaler2.code
```

Comment:
This logic seems to be transferring code/freq from signaler 2 to signaler 1, but the text suggests the opposite.